doc/project/release_process.rst

```diff
@@ -109,6 +109,11 @@ gate the final release. The following counts shall be used:
    The "low" bug count target of <50 will be a phased approach starting with 150
    for release 2.4.0, 100 for release 2.5.0, and 50 for release 2.6.0
 
+The final release must not contain any static analysis high-critical
+issues that can potentially compromise the functionality, security, or
+reliability of our software.  High-critical issues represent
+vulnerabilities that, if left unresolved, could have severe
+consequences.
```

Comment:
would recommend you check your formatter, it looks like it wraps at 72 characters which seems overly aggressive -- 100 would be better. Arguably, the file is currently pretty inconsistently formatted with wrapping a bit all over the place.